kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -153,6 +206,14 @@ define([
                 },
             });
 
+            /**
+             * This is used by the newObjectInput widget to determine whether there's
+             * any duplicated parameters. Since we, effectively, have a sequence of the same
+             * input ids here, this gets tricky.
+             * TODO: deal with trickiness and make this return any values that might be
+             * duplicate. Note, might have to twiddle appWidgets2/input/newObjectInput.js
+             * around line 125. It expects a key-value-pair set of parameters.
+             */
```

Comment:
Still a TODO. The output object names should know to be unique and throw an error otherwise, and this will deal with that.